spec/models/bank_account_spec.rb

```diff
@@ -33,4 +33,25 @@
       expect(bank_account.display_name).to eq 'Test Bank Acct 123456789'
     end
   end
+
+  describe '#benefits ' do
+    it 'returns true if benefits present' do
```

Comment:
Is it worth adding a test for `'returns false if benefits not present'`, to be consistent with the `tax_credits` spec below?

Reply:
Added